MapboxNavigationTests/ImageDownloaderTests.swift

```diff
@@ -39,7 +40,7 @@ class ImageDownloaderTests: XCTestCase {
         var errorReturned: Error?
         let semaphore = DispatchSemaphore(value: 0)
 
-        downloader.downloadImage(with: imageURL) { (image, data, error) in
+        downloader?.downloadImage(with: imageURL) { (image, data, error) in
```

Comment:
Should this test assert that `downloader` is non-`nil`, as the other tests do below?

Reply:
I guess I was being terse since this method only touches it once. Preference was for a runtime failure (nil) vs a crash (unexpected failure unwrapping).

Happy to make it consistent with the other methods before merging.